docs/versioned-plugins/inputs/stdin-v3.2.3.asciidoc

```diff
@@ -28,12 +28,8 @@ want to join lines, you'll want to use the multiline codec.
 [id="{version}-plugins-{type}s-{plugin}-options"]
 ==== Stdin Input Configuration Options
 
-This plugin supports the following configuration options plus the <<{version}-plugins-{type}s-{plugin}-common-options>> described later.
-
-[cols="<,<,<",options="header",]
-|=======================================================================
-|Setting |Input type|Required
-|=======================================================================
+There are no special configuration options for this plugin,
```

Comment:
Removed blank table that causes doc build errors and replaced it with the text we used in later versions of the plugin.